lib/src/main/java/com/auth0/android/lock/internal/configuration/PasswordComplexity.java

```diff
@@ -0,0 +1,34 @@
+package com.auth0.android.lock.internal.configuration;
+
+import android.support.annotation.Nullable;
+
+public class PasswordComplexity {
+
+    private final int policy;
+    private final Integer minLengthOverride;
+
+    public PasswordComplexity(@PasswordStrength int policy, @Nullable Integer minLengthOverride) {
+        this.policy = policy;
+        this.minLengthOverride = minLengthOverride;
+    }
+
+    /**
+     * Getter for the Password Policy associated to this connection.
+     *
+     * @return The Password Policy level for this connection.
+     */
+    @PasswordStrength
+    public int getPasswordPolicy() {
+        return policy;
+    }
+
+    /**
+     * Getter for the minimum length the password requires
+     *
+     * @return the minimum length the password requires
```

Comment:
Should doc comments like this start with a capital letter or not? You're mixing case in this class.

Reply:
when talking about params or return values I rather start with a lower case.